config.json

```diff
@@ -1378,6 +1378,468 @@
+        "blurb": ""
+    },
+    {
+        "uuid": "0649f216-c720-4653-a33e-64b5513b9780",
+        "slug": "methods",
+        "name": "Methods",
+        "blurb": ""
+    },
+    {
+        "uuid": "c61e971c-15b0-4b80-a724-35a841b68720",
+        "slug": "move-semantics",
+        "name": "Move semantics",
+        "blurb": ""
+    },
+    {
+        "uuid": "38785236-be81-44ef-8625-c4054d5d3641",
+        "slug": "mutability",
+        "name": "Mutability",
+        "blurb": ""
+    },
+    {
+        "uuid": "fc6fe604-01b7-476c-86dc-125b441b1fb5",
+        "slug": "mutex",
+        "name": "Mutex",
+        "blurb": ""
+    },
+    {
+        "uuid": "ab78652d-bfc4-4741-b68c-5515a2e1f511",
+        "slug": "newtype-pattern",
+        "name": "Newtype pattern",
+        "blurb": ""
+    },
+    {
+        "uuid": "8fe3f7c1-15d2-434f-8cd3-16b6f410baa5",
+        "slug": "numeric-traits",
+        "name": "Numeric traits",
+        "blurb": ""
+    },
+    {
+        "uuid": "8e7fd01b-b4d6-4386-8259-7817374ce620",
+        "slug": "overflow-and-underflow",
+        "name": "Overflow and underflow",
+        "blurb": ""
+    },
+    {
+        "uuid": "7b456eec-5348-45e5-a442-bcdcc48fda61",
+        "slug": "ranges",
+        "name": "Ranges",
+        "blurb": ""
+    },
+    {
+        "uuid": "e06b4e87-58b2-4dd2-88cd-60f4178ebd54",
+        "slug": "rc",
+        "name": "Rc",
+        "blurb": ""
+    },
+    {
+        "uuid": "3e01401e-1627-4423-aa8b-f9cf52c4adaa",
+        "slug": "references",
+        "name": "References",
+        "blurb": ""
+    },
+    {
+        "uuid": "5a1896c8-b479-4bc4-89e7-a74d34056bde",
+        "slug": "result",
+        "name": "Result",
+        "blurb": ""
+    },
+    {
+        "uuid": "5173a1e4-5909-4dd3-a348-1750c096f1de",
+        "slug": "rwlock",
+        "name": "Rwlock",
+        "blurb": ""
+    },
+    {
+        "uuid": "4ce100e4-8ead-4d9e-b6e2-76cfd4c376f0",
+        "slug": "scopes-and-expressions",
+        "name": "Scopes and expressions",
+        "blurb": ""
+    },
+    {
+        "uuid": "d844d9b8-47e2-40e8-ab12-47a0a3565882",
+        "slug": "shadowing",
+        "name": "Shadowing",
+        "blurb": ""
+    },
+    {
+        "uuid": "7e826a22-45ed-4847-8de9-594bc57daa65",
+        "slug": "slices",
+        "name": "Slices",
+        "blurb": ""
+    },
+    {
+        "uuid": "ac37efdf-a1c5-4526-bd7a-fe1c4cba1656",
+        "slug": "static-lifetime",
+        "name": "Static lifetime",
+        "blurb": ""
+    },
+    {
+        "uuid": "07a32f06-1268-4f17-abbe-7b9c7fcc7995",
+        "slug": "std-thread",
+        "name": "Std thread",
+        "blurb": ""
+    },
+    {
+        "uuid": "fddea778-dba4-4f01-a76c-24bb425686d9",
+        "slug": "string",
+        "name": "String",
+        "blurb": ""
+    },
+    {
+        "uuid": "9fe8a452-a566-4910-807b-6a0fb69265ce",
+        "slug": "string-vs-str",
+        "name": "String vs str",
+        "blurb": ""
+    },
```

Comment:
```suggestion
```

Pretty sure we already have those concepts implemented.

Reply:
Looks like we have string-use but not string-vs-str. That said, I want to resolve #1057 so I will add it to `config.json` when I do that soonish!